packages/cadl-ranch-specs/http/type/property/value-types/main.tsp

```diff
@@ -144,13 +144,13 @@ enum FixedInnerEnum {
 model EnumProperty is ModelTemplate<FixedInnerEnum>;
 @route("/enum")
 @operationGroup
-interface Enum extends ModelOperations<EnumProperty, "ValueOne"> {}
+interface Enum extends ModelOperations<EnumProperty, "\"ValueOne\""> {}
```

Comment:
Does the change only influence summary.md?

Reply:
Yes, this part on `ValueOne` and `UnknownValue` is only for the fix on summary doc.